### ISSUE TITLE
Build failed on master 1.41.0-nightly (abd69551b 2019-11-22)

### ISSUE THREAD

**Issue by yerke** [rust-lang/rust-clippy#4835]
OS: Mac

```
error[E0023]: this pattern has 3 fields, but the corresponding tuple variant has 1 field
    --> clippy_lints/src/utils/author.rs:461:13
     |
461  |             ExprKind::InlineAsm(_, ref _input, ref _output) => {
     |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected 1 field, found 3

error[E0023]: this pattern has 3 fields, but the corresponding tuple variant has 1 field
    --> clippy_lints/src/utils/inspector.rs:285:9
     |
285  |         hir::ExprKind::InlineAsm(_, ref input, ref output) => {
     |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected 1 field, found 3

error[E0023]: this pattern has 3 fields, but the corresponding tuple variant has 1 field
    --> clippy_lints/src/loops.rs:723:11
     |
723  |         | ExprKind::InlineAsm(_, _, _)
     |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected 1 field, found 3

error[E0599]: no method named `parse_str` found for type `rustc_parse::parser::Parser<'_>` in the current scope
   --> clippy_lints/src/write.rs:345:43
    |
345 |     let (fmtstr, fmtstyle) = match parser.parse_str().map_err(|mut err| err.cancel()) {
    |                                           ^^^^^^^^^ help: there is a method with a similar name: `parse_stmt`

error: aborting due to 4 previous errors

Some errors have detailed explanations: E0023, E0599.
For more information about an error, try `rustc --explain E0023`.
error: could not compile `clippy_lints`.
```

**Comment by flip1995** [comment]
Will be fixed with 1b2f43d, once #4825 is merged

**Comment by yerke** [comment]
Thanks for taking a look @flip1995
